src/medley/core.cljc

```diff
@@ -606,6 +640,11 @@
                 (if (compare-and-set! atom value (f value))
                   value
                   (recur))))
+      :cljr (loop []
+              (let [value @atom]
+                (if (compare-and-set! atom value (f value))
+                  value
+                  (recur))))
```

Comment:
I think where the `:clj` and `:cljr` versions are the same, it's probably better to use `:default`.